### ISSUE TITLE
Transaction list: allow to hide categories icons

### ISSUE THREAD

**Issue by Matth7878** [mtotschnig/MyExpenses#1061]
In the new transaction list the left column is used for categories icons. It would be nice to have an option to hide it.
Especially if one have no interest in assigning icons to all categories or prefer to not use horizontal space in favor or the transaction info.

**Comment by mtotschnig** [comment]
@Matth7878 I understand that the new design is currently not adapted to the case where only few transactions are mapped to categories with icons. Out of interest, if you do not need category icons, what would be missing for you, if you switch to the legacy layout?

**Comment by Matth7878** [comment]
When switching to legacy layout you lose improvement in how transaction are now displayed on 2 lines.

For transparency, I reported it but I am going to use new list. I took time to set icons to my main categories (waiting for issue about child without icons to be resolved). Still I thinl some people won't assign categories icons and without them list is weird with just a "-" as a default icon.

Instead of a switch for legacy layout maybe it should be a preferences to customize transaction list : 
- compact mode : to display transaction on a single line with date shortened on left side
- display category icons : on compact mode icons could still be displayed between date and state checkbox. Or it could be not available as an option when using compact mode

If you don't want to change it in my opinion compact mode would be a better name for legacy list. 😉
